scripts/create-secrets/k8sExecution.js

```diff
@@ -0,0 +1,320 @@
+const childProcess = require("child_process");
+const process = require("process");
+const chalk = require("chalk");
+const trim = require("lodash/trim");
+const {
+    getEnvVarInfo,
+    askIfCreateNamespace,
+    settingNameToEnvVarName
+} = require("./askQuestions");
+const Base64 = require("js-base64").Base64;
+const pwgen = require("./pwgen");
+
+const dbPasswordNames = [
+    "authorization-db",
+    "authorization-db-client",
+    "combined-db",
+    "combined-db-client",
+    "discussions-db",
+    "discussions-db-client",
+    "registry-db",
+    "registry-db-client",
+    "session-db",
+    "session-db-client"
+];
+
+let env = process.env;
```

Comment:
Is it possible to keep this off the global namespace and use whatever kind of env is needed at the time? Having it here as a `let` and setting it as a side effect means when I read the code I'm never sure what env currently is.

Reply:
Will pass it as a function parameter to all functions need it~ 👍